.travis.yml

```diff
@@ -17,6 +17,8 @@ gemfile:
 
 matrix:
   exclude:
+    - rvm: 2.2.3
+      gemfile: Gemfile.rails5
```

Comment:
we need to do a little change regarding travis exclude Gemfile.rails5 for rvm 2.2.3 else it gives error, previously this is suggested by your team.